docs/migration/1XX-to-2XX.md

```diff
@@ -24,21 +24,36 @@ for details on how to upgrade to the Firebase `3.x.x` SDK.
 
 ## `$firebaseAuth` Method Renames / Signature Changes
 
+The `$firebaseAuth` service now accepts an optional Firebase `auth` instance instead of a Firebase
```

Comment:
accepts a `firebase.auth.Auth` instance instead of a `firebase.database.Database` instance?